config/webhook/manifests.yaml

```diff
@@ -0,0 +1,30 @@
+
+---
```

Comment:
This should be added to the `static.yaml` in the TCE as well. Similar case is Antrea validation webhook: https://github.com/vmware-tanzu/community-edition/blob/9baddd59e122b8513a8d088495f2963a6a34c7b1/addons/packages/antrea/1.2.3/bundle/config/upstream/antrea.yaml#L4634